src/components/MusicMarkdownNavbar.js

```diff
@@ -36,16 +37,22 @@ const RepositoriesNavDropdown = () => {
   const repoDropdownItems = [];
   const repoList = getRepositories();
   if (repoList.length > 0) {
-    repoList.forEach(function(repo) {
+    repoList.forEach((repo) => {
       const repoId = `${repo.owner}/${repo.repo}${repo.path}`;
       // TODO: List valid files after clicking on repo name
-      const itemHref = `/repos/${repo.owner}/${repo.repo}/contents${repo.path}`;
+      const queryParams = {
+        'owner': repo.owner,
+        'repo': repo.repo,
+        'path': repo.path
+      };
+      const itemHref = `/repo?${queryString.stringify(queryParams)}`;
```

Comment:
This assumes a structure of the URI. Depending on the router, this may not be the appropriate order of components parts. For example, with the hash router, query should come before the fragment.

We should use something like [build-url](https://github.com/steverydz/build-url#readme) to properly construct the URI. [Example of query + fragment](https://github.com/steverydz/build-url#options).

Specifically, this breaks `history` when adding the query after the fragment:

```
> history.location
hash: "#/repos/music-markdown/almost-in-time/contents/...?test=1"
pathname: "/"
search: ""
state: undefined
```